admin/client/src/state/schema/SchemaReducer.js

```diff
@@ -10,6 +11,16 @@ export default function schemaReducer(state = initialState, action = null) {
       return deepFreeze(Object.assign({}, state, { [action.payload.id]: action.payload }));
     }
 
+    case ACTION_TYPES.CLEAR_MESSAGE: {
```

Comment:
I'm not sure we need that - the `state` key in schema is basically considered out-of-date as soon as the form is displayed - form field values, form field validations, and form messages. Since `reduxForm()` is used with `enableReinitialize=false`, you can't propagate form field value changes from schema state to an existing form (and shouldn't need to).

If you carry this reducer-clearing approach through to form fields, you'd change the schema every time a form field validation error is resolved - which causes the whole form `render()` process to kick off (since the `reduxForm()` inputs through `mapStateToProps` changed). This is different from redux-form validation, which only calls `render()` in the individual field.

We could use `setState()` in the `fetch()` response of `FormBuilderLoader` for `state` instead? Or are we just accepting those `render()` calls?